clang/lib/CIR/CodeGen/CIRGenModule.cpp

```diff
@@ -364,19 +364,22 @@ bool CIRGenModule::MayBeEmittedEagerly(const ValueDecl *Global) {
   return true;
 }
 
-static bool hasDefaultVisibility(CIRGlobalValueInterface GV) {
-  // TODO: we need to have a precise definition of what is a default visibility.
-  // in the context of MILR and CIR, now we default to
-  assert(!MissingFeatures::setDefaultVisibility());
-  return true;
+static bool hasDefaultVisibilityForDsoLocal(CIRGlobalValueInterface GV) {
```

Comment:
Please follow OG codegen skeleton, I don't see a good reason for the name change here, or at least make it a helper but keep the entry point `hasDefaultVisibility`